CONTRIBUTING.md

```diff
@@ -226,6 +226,7 @@ When developing new features it can be helpful to enable logging. By setting `CD
 9. Run the [prebuilt provider upgrade workflow](https://github.com/hashicorp/cdktf-repository-manager/actions/workflows/upgrade-repositories.yml)
 10. Update the learn examples and the end to end examples
 11. Check if there are PRs left behind on our [triage board](https://github.com/orgs/hashicorp/projects/125/views/4)
+12. After the release is before the release, create a PR with a heading for the next release (e.g. `## 0.12.0 (unreleased)`) with a `**Breaking changes**` section.
```

Comment:
I'm not quite sure what this is trying to say.

Reply:
I tried to improve the wording a bit